src/integrations/prefect-aws/tests/test_batch.py

```diff
@@ -74,8 +84,45 @@ def test_flow():
 
     job_id = test_flow()
 
-    try:
-        UUID(str(job_id))
-        assert True, f"{job_id} is a valid UUID"
-    except ValueError:
-        assert False, f"{job_id} is not a valid UUID"
+    assert_valid_job_id(job_id)
+
+
+async def test_batch_submit_async_dispatch(
+    job_queue_arn, job_definition_arn, aws_credentials
+):
+    @flow
+    async def test_flow():
+        return await batch_submit(
+            "batch_test_job",
+            job_queue_arn,
+            job_definition_arn,
+            aws_credentials,
+        )
+
+    job_id = await test_flow()
+    assert_valid_job_id(job_id)
+
+
+async def test_batch_submit_explicit_async(
+    job_queue_arn, job_definition_arn, aws_credentials
+):
+    job_id = await batch_submit_async(
+        "batch_test_job",
+        job_queue_arn,
+        job_definition_arn,
+        aws_credentials,
+    )
+    assert_valid_job_id(job_id)
+
+
+async def test_batch_submit_force_sync_from_async(
+    job_queue_arn, job_definition_arn, aws_credentials
+):
+    job_id = batch_submit(
+        "batch_test_job",
+        job_queue_arn,
+        job_definition_arn,
+        aws_credentials,
+        _sync=True,
+    )
+    assert_valid_job_id(job_id)
```

Comment:
i tried a couple permutations of pytest tricks but explicitly writing out the individual cases felt simplest to me

Reply:
The smallest of nits, but can you group the tests into classes so it's easier to find the tests for each implementation?